lib/rules/no-disallowed-property-values.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'no-disallowed-property-values',
+  'defaults': {
+    'properties': {}
+  },
+  'detect': function (ast, parser) {
+    var result = [];
+
+    ast.traverseByType('value', function (valueNode, idx, parent) {
+      var propertyNode = parent.first('property');
+      if (!propertyNode) {
+        return;
+      }
+      var propertyNameNode = propertyNode.first();
+      if (!propertyNameNode.is('ident')) {
+        return;
+      }
+      var propertyName = propertyNameNode.content;
+      var disallowedPropertyValues = parser.options.properties[propertyName];
+      if (!disallowedPropertyValues) {
+        return;
+      }
+      if (typeof disallowedPropertyValues === 'string') {
```

Comment:
You allow property values to be specified as strings outside of arrays here, you don't have this in your tests, it would be good to add this case in. Maybe duplicate the ` display: ['inline'] ` as just `display: 'inline'` that would help.

Reply:
Done! Though I ended up just replacing the `display: ['inline']` case with `display: 'inline'` which feels like it should be sufficient?